railgun/internal/manager/run.go

```diff
@@ -9,44 +9,39 @@ import (
 
 	"github.com/bombsimon/logrusr"
 	"github.com/go-logr/logr"
+	"github.com/kong/deck/file"
 	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
-
+	k8scache "k8s.io/client-go/tools/cache"
+	knativeversioned "knative.dev/networking/pkg/client/clientset/versioned"
+	knativeinformerexternal "knative.dev/networking/pkg/client/informers/externalversions"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/cache"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 
-	"github.com/kong/deck/file"
 	"github.com/kong/kubernetes-ingress-controller/pkg/sendconfig"
 	"github.com/kong/kubernetes-ingress-controller/pkg/util"
-
 	konghqcomv1 "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1"
 	configurationv1beta1 "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1beta1"
-	"github.com/kong/kubernetes-ingress-controller/railgun/controllers/configuration"
-	kongctrl "github.com/kong/kubernetes-ingress-controller/railgun/controllers/configuration"
+	kongctrl "github.com/kong/kubernetes-ingress-controller/railgun/internal/controllers/configuration"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#imports

Please avoid renaming imports unless it resolves a naming conflict.
Prefer renaming packages to aliasing imports. (Discussed out of band)

Reply:
I've removed this import alias in the most recent update